autopush/tests/test_integration.py

```diff
@@ -1005,7 +1005,7 @@ def test_ttl_batch_expired_and_good_one(self):
         data2 = str(uuid.uuid4())
         client = yield self.quick_register()
         yield client.disconnect()
-        for x in range(0, 12):
```

Comment:
was this change necessary here? IIRC the total messages churned through in these batch tests was > 10 to thoroughly test the fetch_messages call with its default limit=10

Reply:
yes, the problem was that we would load up with the 10 expired messages and never get to the one that wasn't expired. That's highlighted now that we're actually expiring messages. The next test checks batching.